resources/user-guides/app-extension.md

```diff
@@ -0,0 +1,74 @@
+# iOS App Extension Implementation
+
+You can use supported AEP extensions in your App Extensions to collect usage data. Supported Adobe Extensions are shown [here](https://aep-sdks.gitbook.io/docs/resources/upgrading-to-aep/current-sdk-versions#ios-swift). This tutorial assumes a basic understanding of how to use the iOS SDK in applications.
```

Comment:
"You can use supported Platform extensions in your App Extensions to collect usage data."

Additionally, use a relative link instead of directly linking to the gitbook, since it's better for future compatibility.